Formula/f3d.rb

```diff
@@ -0,0 +1,39 @@
+class F3d < Formula
+  desc "Fast and minimalist 3D viewer"
+  homepage "https://kitware.github.io/F3D/"
+  url "https://gitlab.kitware.com/f3d/f3d/-/archive/v1.0.1/f3d-v1.0.1.tar.gz"
+  sha256 "fb362dba3ccf49db9e8841d8a5310f37399bfea8866b6e040ce85670d54b97f7"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "vtk"
+
+  def install
+    # Is fixed upstream, remove this line for v1.1
+    inreplace "src/F3DOptions.cxx", "/etc/f3d/", "/usr/local/etc/f3d/"
+
+    args = std_cmake_args + %W[
+      -DCMAKE_BUILD_TYPE:STRING=Release
+      -DMACOSX_BUILD_BUNDLE:BOOL=OFF
+      -DBUILD_SHARED_LIBS:BOOL=ON
+      -DBUILD_TESTING:BOOL=OFF
+      -DF3D_INSTALL_DEFAULT_CONFIGURATION_FILE:BOOL=ON
+      -DCMAKE_INSTALL_NAME_DIR:STRING=#{lib}
+      -DCMAKE_INSTALL_RPATH:STRING=#{lib}
+    ]
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    mkdir_p prefix/"f3d"
+    mv prefix/"config.json", prefix/"f3d/config.json"
+    etc.install_symlink prefix/"f3d"
```

Comment:
If this should be user configurable it shouldn't be symlinked because an update would override it then.